src/eureka/S3_data_reduction/background.py

```diff
@@ -281,14 +281,14 @@ def fitbg(dataim, meta, mask, x1, x2, deg=1, threshold=5, isrotate=False,
             # background image
             if len(goodxvals) != 0:
                 bg[j] = np.polyval(coeffs, range(nx))
-                if isplots >= 6:
+                if isplots == 6:
```

Comment:
Why not `>=`? That's how all the other isplots are handled, right?

Reply:
I was testing parts of the code that required isplots > 6 and didn't want to sit and wait for the code to create lots of plots that I didn't care about.  In general, it makes sense to use `==` for values of 6+ because people won't be investigating different parts of the code at the same time.